internal/pkg/service/templates/api/service/mapper.go

```diff
@@ -130,6 +134,25 @@ func TemplatesResponse(ctx context.Context, d dependencies.ProjectRequestScope,
 			continue
 		}
 
+		if filterBy != nil && *filterBy != "" {
+			filterString := *filterBy
+
+			t, found := tmpl.DefaultVersion()
+			if !found {
+				continue
+			}
+			switch filterString {
+			case keboola.DataAppsComponentID.String():
```

Comment:
I would rather define new constant here. The `go-client` is not the best place to get constants from. It is keboola specific thing that can change due to some API change and therefore the UI would need to update also templates pages.

Reply:
[0a285f9](https://github.com/keboola/keboola-as-code/pull/2144/commits/0a285f96b192a0fe48f094e048828c14095fbcc5)